guides/common/modules/con_user-groups.adoc

```diff
@@ -3,6 +3,6 @@
 
 With {Project}, you can assign permissions to groups of users.
 You can also create user groups as collections of other user groups.
-If using an external authentication source, you can map {Project} user groups to external user groups as described in xref:Configuring_External_User_Groups_{context}[].
+If using an external authentication source, you can map {Project} user groups to external user groups as described in {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}[Configuring external user groups] in _{InstallingServerDocTitle}_.
```

Comment:
```suggestion
If using an external authentication source, you can map {Project} user groups to external user groups as described in {InstallingServerDocURL}Configuring_External_User_Groups_admin[Configuring external user groups] in _{InstallingServerDocTitle}_.
```

IMHO, xrefs to another document always require the hardcoded context from the guide itself. See line 3: https://raw.githubusercontent.com/theforeman/foreman-documentation/master/guides/doc-Administering_Project/master.adoc

Reply:
If a specific section is only part of one build target (for example Satellite), then yes, you'll need `ifdef::satellite[]` for the xref in any other file.

I've looked at the wrong guide :see_no_evil: 
You're absolutely right, it should be `{project-context}`: https://github.com/theforeman/foreman-documentation/blame/master/guides/doc-Installing_Server/master.adoc#L4